components/director/internal/tenantfetchersvc/provisioner.go

```diff
@@ -12,121 +11,112 @@ import (
 	"github.com/pkg/errors"
 )
 
-const regionLabelKey = "region"
+const autogeneratedTenantProvider = "autogenerated"
```

Comment:
What is the purpose of using "autogenerated" for Customers?

Reply:
In case we start gathering more info for the customer tenants (for example we get notified when one is created, just like with accounts), then we'll know which tenants should be updated and "enriched"